app/src/main/java/org/greenstand/android/TreeTracker/di/AppModule.kt

```diff
@@ -56,18 +82,18 @@ val appModule = module {
 
     single { androidContext().getSystemService(Context.LOCATION_SERVICE) as LocationManager }
 
-    single { UserLocationManager(get(), get()) }
+    single { LocationUpdateManager(get(), get()) }
 
     single { ObjectStorageClient.instance() }
 
     single { NotificationManagerCompat.from(get()) }
 
     single {
-        CaptureTreeLocationUseCase(
-                get(),
-                get(),
-                get()
-            )
+        LocationDataCapturer(
```

Comment:
Renamed the `CaptureTreeLocationUseCase` to `LocationDataCapturer` and moved it out of the usecases package and into LocationUpdateManager.  It wasn't inheriting from the `UseCase` class anyways and I don't feel like it is a usecase.
Like in my other observations, `UseCase`  for me is a coordination of objects to perform a process or function at a higher level.